examples/arctic_basin_seasonal_cycle.jl

```diff
@@ -1,54 +1,86 @@
-#####
-##### The purpose of this script is to reproduce some results from Semtner (1976)
-#####
+# # Arctic basin seasonal cycle example
+#
+# This example reproduces results from Semtner (1976), which simulates the seasonal
+# cycle of sea ice in the Arctic basin. The model uses climatological forcing data
+# from Fletcher (1965) for shortwave radiation, longwave radiation, sensible heat
+# flux, and latent heat flux. This example demonstrates
+#
+#   * How to use time-varying climatological forcing data.
+#   * How to set up seasonal cycles with `FieldTimeSeries` and cyclical indexing.
+#   * How to combine multiple heat flux components.
+#   * How to simulate multi-year seasonal cycles.
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+#
+# ```julia
+# using Pkg
+# pkg"add Oceananigans, ClimaSeaIce, CairoMakie"
+# ```
+#
+# ## The physical domain
+#
+# We use a single grid point to model a horizontally uniform ice column:
 
 using Oceananigans
 using Oceananigans.Units
 using Oceananigans.Utils: Time
 using Oceananigans.OutputReaders
```

Comment:
we shouldn't have to import this

something I put in the AGENTS.md file for Oceananigans was to minimize import statements in examples. Examples should exercise the public user interface, which is exported by `using ClimaSeaIce`.

We could also reexport the relevant parts of the Oceananigans interface by ClimaSeaIce directly. This is the approach I am taking with Breeze, so that people only need to write `using Breeze`.